app/src/main/java/com/keylesspalace/tusky/BaseActivity.java

```diff
@@ -212,6 +212,13 @@ public void showAccountChooserDialog(CharSequence dialogTitle, boolean showActiv
         }
     }
 
+    // TODO: This changes the accountManager's activeAccount property, but does not do any
+    // of the work that AccountManager.setActiveAccount() does. In particular:
+    //
+    // - The current active account is not saved
+    // - The account passed as parameter here goes not have its `isActive` property set
+    //
+    // Is that deliberate? Or is this a bug?
```

Comment:
I think it is a bug. Does not really make sense this way.

Reply:
OK. I've filed https://github.com/tuskyapp/Tusky/issues/3702 to track looking in to that in more detail after v22 is released.